src/lib/components/DamageMeter.svelte

```diff
@@ -204,9 +204,29 @@
                 if (duration < 0) {
                     encounterDuration = millisToMinutesAndSeconds(0);
                     dps = 0;
+                    timeUntilKill = "00:00";
                 } else {
                     encounterDuration = millisToMinutesAndSeconds(duration);
                     dps = totalDamageDealt / (duration / 1000);
+                    if ($settings.meter.timeUntilKill) {
```

Comment:
`if (encounter.currentBoss && $settings.meter.timeUntilKill) {`

Reply:
The check is in but I think it does not change anything since [these lines](https://github.com/JonasHri/loa-logs/blob/7f0336374de3025073c1091af145573427c50dbb/src/lib/components/DamageMeter.svelte#L219-L224) already cover a boss not existing.